syft/messaging/message.py

```diff
@@ -8,9 +8,8 @@
 """
 
 import syft as sy
-
-from syft.workers import AbstractWorker
 from syft import codes
+from syft.workers import AbstractWorker
```

Comment:
So glad you changed the order of these two imports to be non-alphabetical. Hardcore bothered me last night.

Reply:
🙂